src/main/java/org/eolang/EOarray.java

```diff
@@ -0,0 +1,103 @@
+package org.eolang;
+
+import org.eolang.core.EOObject;
+import org.eolang.core.EOObjectArray;
+import org.eolang.core.data.EOData;
+import org.eolang.core.data.EODataObject;
+
+public class EOarray extends EOObject {
+    private EOObjectArray array;
+
+    public EOarray(EOObjectArray array){
+        this.array = array;
+    }
+
+    @Override
+    public EOData _getData() {
+        return array._getData();
+    }
+
+    /***
+     * Determines if the base array is empty
+     * @return A boolean object, true if empty and false if not empty
+     */
+    public EObool isEmpty(){
+        return new EObool(array.get_array().length == 0);
+    }
+
+    /***
+     * Gets the length of the array
+     * @return An object representing the length of the array
+     */
+    public EOint length(){
+        return new EOint(array.get_array().length);
+    }
+
+    /***
+     * Get the object at (@code position) free attribute of the array
```

Comment:
`{@code}`. It may not work with parentheses. 
If you use IDEA, please, have a look at the left upper corner above the JAVADOC comment. There should be a tiny icon to preview JAVADOC. there you can see how it would be rendered.